packages/js/src/utils/id-token.ts

```diff
@@ -0,0 +1,39 @@
+import { UrlSafeBase64 } from '@silverhand/essentials';
+import * as s from 'superstruct';
+
+/**
+ * @link [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken)
+ */
+const IdTokenClaimsSchema = s.type({
+  iss: s.string(),
+  sub: s.string(),
+  aud: s.string(),
+  exp: s.number(),
+  iat: s.number(),
+  at_hash: s.optional(s.string()),
+});
+
+export type IdTokenClaims = s.Infer<typeof IdTokenClaimsSchema>;
+
+export const decodeIdToken = (token: string): IdTokenClaims => {
+  const { 1: encodedPayload } = token.split('.');
```

Comment:
nit:
```js
const [ _, encodedPayload ] = token.split('.');
```

Reply:
eslint doesn't recommend me to use code above.

![image](https://user-images.githubusercontent.com/10594507/150090856-c0b558da-4a9e-4ebc-861b-c46d1479263c.png)

Reply:
https://files.slack.com/files-pri/T0243NVUC9E-F02UZQYRQJV/image.png

Reply:
![image](https://user-images.githubusercontent.com/10594507/150091773-153ace83-d1f4-49d9-a733-25909cfe38ce.png)